src/main/java/com/example/enterprise/service/IncomeService.java

```diff
@@ -31,7 +31,7 @@ public void save(Income income) {
     }
 
     @Override
-    public List<Income> listAll() {
+    public List<Income> listAllIncomes() {
```

Comment:
Changed the method name from listAll() to listAllIncomes() because listAll doesn't specify what to list. An outsider would not understand what it is trying to list. listAllIncomes() name is much more meaningful and provides more clarity to the code logic.